app/views/content_items/_detailed_guide_b.html.erb

```diff
@@ -0,0 +1,54 @@
+<%= content_for :title, @content_item.page_title %>
```

Comment:
Or the logic could be in the partial itself, if it affects all breadcrumbs on all formats with breadcrumbs.